### ISSUE TITLE
I'll pay for updating the library (PROXY)

### ISSUE THREAD

**Issue by skvoshiz** [sochix/TLSharp#596]
This Lib supports only Socks IPv4
https://github.com/sochix/TLSharp/issues/593
I need use **HTTP(S)/Socks IPv4 and IPv6**
How much will it cost?

**Comment by Troodi** [comment]
For what?
```C#
TcpTransport transport;
transport = new TcpTransport(ip, port, handle);
``` 
Use socks ip:port, works for me.

**Comment by skvoshiz** [comment]
Its work only for IPv4, i need IPv6

**Comment by knocte** [comment]
Fixed in 3224113e4ce827319c049f44ee5a759ad1591fa0
